### DISCUSSION TITLE
Getting 'Cannot read property 'logEvent' of undefined' error once I clone and run Hoppscotch locally

### DISCUSSION THREAD

**Discussion by ozsoft** [hoppscotch/hoppscotch#1754] [Q&A] [answered]
Any ideas why I get this error:

![image](https://user-images.githubusercontent.com/20835364/127866107-5b7ee62f-2629-49ba-9695-55e03e27f2a8.png)

**Comment by liyasthomas** [comment, marked as answer]
If you're running locally, please find the [full development setup steps](https://github.com/hoppscotch/hoppscotch#developing).

Did you renamed the `.env.example` file found in root directory to `.env`?

**Reply by ozsoft** [reply]
Thanks Liyas, it was the renaming indeed!
